docs/docs/mlflow-3/index.mdx

```diff
@@ -14,7 +14,7 @@ Discover the next generation of MLflow, designed to streamline your AI experimen
 ## What is MLflow 3.0?
 
 The best AI experimentation platform for building high quality models and GenAI agents! MLflow 3.0 introduces a variety of exciting improvements to MLflow Tracking and Tracing, including:
-- First class model support in MLflow tracking with **LoggedModel**
+- First class model support in MLflow tracking with <APILink fn="mlflow.entities.LoggedModel">**LoggedModel**</APILink>
 - **Model-centric** information architecture for params and metrics in MLflow tracking
```

Comment:
Can we remove all references to "model-centric" and "information architecture" in our docs? I'd rather talk about how MLflow 3.0 makes it easy to "centrally track all of the performance data for a model across environments"

Reply:
Got it